src/qml/QFieldSettings.qml

```diff
@@ -361,6 +361,173 @@ Page {
+                      onCheckedChanged: {
+                          positioningSettings.showPositionInformation = checked
+                      }
+                  }
+
+                  Label {
+                      text: qsTr("Activate accuracy indicator")
+                      font: Theme.defaultFont
+                      wrapMode: Text.WordWrap
+                      Layout.fillWidth: true
+
+                      MouseArea {
+                          anchors.fill: parent
+                          onClicked: accuracyIndicator.toggle()
+                      }
+                  }
+
+                  QfSwitch {
+                      id: accuracyIndicator
+                      Layout.preferredWidth: implicitContentWidth
+                      Layout.alignment: Qt.AlignTop
+                      checked: positioningSettings.accuracyIndicator
+                      onCheckedChanged: {
+                          positioningSettings.accuracyIndicator = checked
+                      }
+                  }
+
+                  Label {
+                      text: qsTr("Bad accuracy below [m]")
+                      font: Theme.defaultFont
+                      enabled: accuracyIndicator.checked
+                      visible: accuracyIndicator.checked
+                      Layout.leftMargin: 8
+                  }
+
+                  TextField {
+                      id: accuracyBadInput
+                      width: antennaHeightActivated.width
+                      font: Theme.defaultFont
+                      enabled: accuracyIndicator.checked
+                      visible: accuracyIndicator.checked
+                      horizontalAlignment: TextInput.AlignHCenter
+                      Layout.preferredWidth: 60
+                      Layout.preferredHeight: font.height + 20
+
+                      inputMethodHints: Qt.ImhFormattedNumbersOnly
+                      validator: DoubleValidator {}
+
+                      background: Rectangle {
+                        y: parent.height - height - parent.bottomPadding / 2
+                        implicitWidth: 120
+                        height: parent.activeFocus ? 2: 1
+                        color: parent.activeFocus ? '#4CAF50' : '#C8E6C9'
+                      }
+
+                      Component.onCompleted: {
+                          text = isNaN( positioningSettings.accuracyBad ) ? '' : positioningSettings.accuracyBad
+                      }
+
+                      onTextChanged: {
+                          if( text.length === 0 || isNaN(text) ) {
+                              positioningSettings.accuracyBad = NaN
+                          } else {
+                              positioningSettings.accuracyBad = parseFloat( text )
+                          }
+                      }
+                  }
+
+                  Label {
+                      text: qsTr("Excellent accuracy above [m]")
```

Comment:
```suggestion
                      text: qsTr("Excellent accuracy %1 above [m]").arg('<font color="#e41a1c">•</font>')
```

What do you think about showing the color here?
Green for excellent
Yellow for acceptable

Reply:
Not a bad idea. Maybe we can show those in the gray explanatory paragraphs instead.